src/Files.App/App.xaml.cs

```diff
@@ -206,7 +206,7 @@ private static async Task InitializeAppComponentsAsync()
                     SidebarPinnedController.InitializeAsync()
                 );
                 await Task.WhenAll(
-                    AppSettings.DetectQuickLook(),
+                    QuickLookHelpers.DetectQuickLook(),
```

Comment:
Would it improve performance to defer this task until the first time the user presses the spacebar?

Reply:
I imagine it would, but only slightly. I'll investigate and get back to you.

Reply:
According to the profiler, it spends about 2ms in the detect method.

Reply:
I think it's worth improving, every additional ms counts on startup especially on slower devices where it can take several times longer.

Reply:
Good point. I’ll make the changes soon.